frontend/src/components/edit-profile/edit-profile.tsx

```diff
@@ -224,7 +224,7 @@ export const EditProfile: React.FC<EditProfileProps> = ({ onClose }) => {
                     <MenuItem value="not_appliable">Rather not say</MenuItem>
                   </SelectFieldForm>
                   <SelectYearRange
-                    start={1960}
+                    start={new Date().getFullYear() - 95}
```

Comment:
This should be in some variable. Firstly - to be more explicit. Secondly, on each rerender a new date object will be created